metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -1165,4 +1192,22 @@ public long curClaimEpoch() {
     public void close() throws InterruptedException {
         queue.close();
     }
+
+    // VisibleForTesting
+    CountDownLatch pause() {
+        final CountDownLatch latch = new CountDownLatch(1);
+        appendControlEvent("pause", () -> {
+            try {
+                latch.await();
+            } catch (InterruptedException e) {
+                log.info("Interrupted while waiting for unpause.", e);
+            }
+        });
+        return latch;
+    }
+
+    // VisibleForTesting
+    Time time() {
+        return time;
+    }
```

Comment:
Do we need this? Can't we just use the Time we pass into the constructor in tests? Not a big deal really, just wondering

Reply:
We have several helper classes for creating quorum controllers, so getting access to the time parameter that way would be difficult.  Anyway, this is package-private, so it really only applies to unit tests specifically.